app/Http/Controllers/User.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class User extends Controller
+{
+    public function users()
```

Comment:
I think this is not yet needed in this PR yet, please remove for now.

Reply:
I left this for next task.
I will write the memo in next branches not to confuge you all.